crates/bevy_ui/src/node_bundles.rs

```diff
@@ -91,8 +91,6 @@ pub struct ImageBundle {
     /// The calculated size based on the given image
     pub calculated_size: ContentSize,
     /// The background color, which serves as a "fill" for this node
-    ///
-    /// Combines with `UiImage` to tint the provided image.
     pub background_color: BackgroundColor,
```

Comment:
Probably not related to this PR but, should that component be optional ?

Reply:
Optional or not included in the bundle itself (that might be what you mean) might make sense. `BackgroundColor` is included in `TextBundle` as well, so this is consistent at least.

Reply:
Optional or not included in the bundle itself (that might be what you mean) might make sense. `BackgroundColor` is included in `TextBundle` as well, so this is consistent at least.

Reply:
Doesn't forcing this component add useless extra computations with its default value or non transparent images ?

Reply:
It defaults to `LegacyColor::NONE` in this bundle, in which case it'll be skipped by `extract_uinode_background_colors`. Technically iterating over it and checking its value before skipping it is useless extra computations... but `TextBundle` does the same thing already before this PR.

Reply:
I'm not sure what you mean by non transparent images. So the background color would be hidden behind the image?

Reply:
@alice-i-cecile After this PR there should probably be a pass on components that should be optional like `TextureAtlas` and `ImageScaleMode`, I think `BackgroundColor` could be considered an _additional_ behaviour component as well

Reply:
Yeah, I'm interested in that too.